### PR TITLE
Doc fixes stable 4.0

### PR DESCRIPTION
**Pull request by ru-fu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,22 +7,26 @@ It offers a unified user experience around full Linux systems running inside con
...
+CI (client)         | GitHub                | [![Build Status](https://github.com/canonical/lxd/workflows/Client%20build%20and%20unit%20tests/badge.svg)](https://github.com/canonical/lxd/actions)
 CI (server)         | Jenkins               | [![Build Status](https://jenkins.linuxcontainers.org/job/lxd-github-commit/badge/icon)](https://jenkins.linuxcontainers.org/job/lxd-github-commit/)
-Go documentation    | Godoc                 | [![GoDoc](https://godoc.org/github.com/lxc/lxd/client?status.svg)](https://godoc.org/github.com/lxc/lxd/client)
-Static analysis     | GoReport              | [![Go Report Card](https://goreportcard.com/badge/github.com/lxc/lxd)](https://goreportcard.com/report/github.com/lxc/lxd)
+Go documentation    | Godoc                 | [![GoDoc](https://godoc.org/github.com/lxc/lxd/client?status.svg)](https://godoc.org/github.com/lcx/lxd/client)
+Static analysis     | GoReport              | [![Go Report Card](https://goreportcard.com/badge/github.com/lcx/lxd)](https://goreportcard.com/report/github.com/lxc/lxd)
```
Comment (tomponline) [inline README.md]: @ru-fu typo here? lcx?

File: README.md [hunk 2]
```diff
@@ -8,22 +7,26 @@ It offers a unified user experience around full Linux systems running inside con
...
 ## Status
 Type                | Service               | Status
 ---                 | ---                   | ---
-CI (client)         | GitHub                | [![Build Status](https://github.com/lxc/lxd/workflows/Client%20build%20and%20unit%20tests/badge.svg?branch=stable-4.0)](https://github.com/lxc/lxd/actions)
+CI (client)         | GitHub                | [![Build Status](https://github.com/canonical/lxd/workflows/Client%20build%20and%20unit%20tests/badge.svg)](https://github.com/canonical/lxd/actions)
 CI (server)         | Jenkins               | [![Build Status](https://jenkins.linuxcontainers.org/job/lxd-github-commit/badge/icon)](https://jenkins.linuxcontainers.org/job/lxd-github-commit/)
-Go documentation    | Godoc                 | [![GoDoc](https://godoc.org/github.com/lxc/lxd/client?status.svg)](https://godoc.org/github.com/lxc/lxd/client)
-Static analysis     | GoReport              | [![Go Report Card](https://goreportcard.com/badge/github.com/lxc/lxd)](https://goreportcard.com/report/github.com/lxc/lxd)
+Go documentation    | Godoc                 | [![GoDoc](https://godoc.org/github.com/lxc/lxd/client?status.svg)](https://godoc.org/github.com/lcx/lxd/client)
```
Comment (tomponline) [inline README.md]: lcx?

File: README.md [hunk 3]
```diff
@@ -8,22 +7,26 @@ It offers a unified user experience around full Linux systems running inside con
...
+- Release announcements: [`https://discourse.ubuntu.com/c/lxd/news/`](https://discourse.ubuntu.com/c/lxd/news/)
+- Release tarballs: [`https://github.com/canonical/lxd/releases/`](https://github.com/canonical/lxd/releases/)
+- Documentation: [`https://documentation.ubuntu.com/lxd/en/stable-4.0/`](https://documentation.ubuntu.com/lxd/en/stable-4.0/)
+
+
 ## Status
 Type                | Service               | Status
 ---                 | ---                   | ---
-CI (client)         | GitHub                | [![Build Status](https://github.com/lxc/lxd/workflows/Client%20build%20and%20unit%20tests/badge.svg?branch=stable-4.0)](https://github.com/lxc/lxd/actions)
+CI (client)         | GitHub                | [![Build Status](https://github.com/canonical/lxd/workflows/Client%20build%20and%20unit%20tests/badge.svg)](https://github.com/canonical/lxd/actions)
```
Comment (tomponline) [inline README.md:26]: Can we target workflows on specific branches for the badge?

File: README.md (hunk 1 above)
Comment (ru-fu) [reply README.md]: outdated

**Pull request merged**